packages/fxa-content-server/app/scripts/lib/router.js

```diff
@@ -206,9 +206,15 @@ Router = Router.extend({
     'legal(/)': function () {
       this.createReactOrBackboneViewHandler('legal', 'legal');
     },
-    'legal/privacy(/)': createViewHandler('pp'),
+    'legal/privacy(/)': function () {
+      this.createReactOrBackboneViewHandler('legal/privacy', 'pp', {
```

Comment:
I think in this ticket we should mention to QA that they should look at this page. I'm not sure how it was missed, I [noticed this earlier](https://github.com/mozilla/fxa/pull/15340#discussion_r1201338291) and thought we'd land it there, so I'm super glad it'll be landing now heh. 👍